web/src/cypress/integration/rotations.ts

```diff
@@ -103,6 +103,17 @@ function testRotations(screen: ScreenFormat) {
       }),
     )
 
+    it('should allow setting and unsetting as a favorite rotation', () => {
```

Comment:
If we move the rest of the favorites tests into one favorites file, this test can be moved there as well as the service's favorites tests